### ISSUE TITLE
Include in website a Try Java section  <niceToHave>

### ISSUE THREAD

**Issue by cesarhernandezgt** [aalmiray/java-champions#112]
It would be nice to have a section like "Try Java" or  "Run Java" or some sort of equivalence to the embedded https://www.python.org/

![image](https://user-images.githubusercontent.com/1454523/211849032-7ddff17b-e1f9-400f-a5b6-a4f13f3363d5.png)

**Comment by cesarhernandezgt** [comment]
The last interactive shell with embeddable options I tried a couple of years ago was katakoda, but it no longer exists. 

Looking at jdoodle, the downside I see is that user can't edit the code when you use the [embedded](https://docs.jdoodle.com/integrating-compiler-ide-to-your-application/embed-a-shared-code#accessing-downloading-a-shared-program) option.

I spined a draft PR using [onecompiler](https://onecompiler.com/) via https://github.com/aalmiray/java-champions/pull/113 for the overall structure. In case we don't have a preview site for PR's, the overall result is this.:

![image](https://user-images.githubusercontent.com/1454523/211876719-247b1be4-2499-4108-9bba-f2f0ae898412.png)

**Comment by aalmiray** [comment]
@cesarhernandezgt I'd suggest discussing the pros and cons at the mailing list first, as the use of this 3rd party website could be seen as a tacit endorsement by the Java Champions group. if that's the case then we'd have to agree on it first.

Also, the `System` class is in the `java.lang` package, not `java.util`. The use of star imports is good for getting started but frowned upon later. I'm sure we'll have many opinions on how the prototypical HelloWorld code should look like for beginners. I'd personal defer to those that teach Java to help us with their advice.

**Comment by cesarhernandezgt** [comment]
@aalmiray I agree with you, I just picked the second google result on "online Java compiler".
Rather than implementation, the question is if there is an opportunity the website can take to offer to users a `Try Java` section.

**Comment by cesarhernandezgt** [comment]
I updated the PR draft to remove text details and left the vanilla hello world code that, out of the box, still has the include line, but as mentioned before, details on the implementation can be tackled later if the Try Java sections reach consensus.

![image](https://user-images.githubusercontent.com/1454523/211889165-3c83c6a0-6ae9-43f5-9e0f-ed745d5022a6.png)
